helm-charts/common/embedding-usvc/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: monitoring.coreos.com/v1
```

Comment:
we need to make this whole service monitor by using the same switch as the HPA definition